### DISCUSSION TITLE
Release schedule

### DISCUSSION THREAD

**Discussion by tr33m4n** [percy/cli#625] [Support] [answered]
Hi there,

I was wondering whether a stable release of https://github.com/percy/cli is on the cards anytime soon? I don't mean to rush you guys, however with https://github.com/percy/percy-agent now being deprecated and my own personal projects flagging security issues with some of the Percy agent dependencies, I feel like I'm stuck between a rock and a hard place! Do I upgrade and risk instability, or do I hang fire and risk security vulnerabilities?

Please don't take this the wrong way, I'm absolutely loving the work so far. If you could advise whether the beta releases are stable enough for production use, that would do me.

Thanks

**Comment by Robdel12** [comment, marked as answer]
Hey @tr33m4n! You can absolutely adopt `@percy/cli` into your production projects -- we're not shipping any breaking changes between now & v1.0.0. You'll see a large stability improvement migrating from agent to CLI, too. 😃 

We _just_ finished migrating all SDKs to use Percy CLI. We were holding off on going v1.0 until that was completed (in case we came across any refactors we wanted to take care of now). We're probably 3 to 4 weeks away from releasing 1.0.0. The last feature we need to ship prior to that is a version check & upgrade notice so we can recommend installing the CLI globally & distribute it on other package managers like `brew`. 

TL;DR: `@percy/cli` is the officially recommend package, I wouldn't recommend staying on `@percy/agent`

**Reply by tr33m4n** [reply]
Excellent, thanks for the info, I'll upgrade ASAP 😄
